GPU/Common/FragmentShaderGenerator.cpp

```diff
@@ -58,23 +72,33 @@ bool GenerateFragmentShader(const FShaderID &id, char *buffer, const ShaderLangu
 
 	ReplaceAlphaType stencilToAlpha = static_cast<ReplaceAlphaType>(id.Bits(FS_BIT_STENCIL_TO_ALPHA, 2));
 
-	std::vector<const char*> gl_exts;
+	std::vector<const char*> extensions;
 	if (ShaderLanguageIsOpenGL(compat.shaderLanguage)) {
 		if (stencilToAlpha == REPLACE_ALPHA_DUALSOURCE && gl_extensions.EXT_blend_func_extended) {
-			gl_exts.push_back("#extension GL_EXT_blend_func_extended : require");
+			extensions.push_back("#extension GL_EXT_blend_func_extended : require");
 		}
 		if (gl_extensions.EXT_gpu_shader4) {
-			gl_exts.push_back("#extension GL_EXT_gpu_shader4 : enable");
+			extensions.push_back("#extension GL_EXT_gpu_shader4 : enable");
 		}
 		if (compat.framebufferFetchExtension) {
-			gl_exts.push_back(compat.framebufferFetchExtension);
+			extensions.push_back(compat.framebufferFetchExtension);
 		}
 		if (gl_extensions.OES_texture_3D && texture3D) {
-			gl_exts.push_back("#extension GL_OES_texture_3D: enable");
+			extensions.push_back("#extension GL_OES_texture_3D: enable");
 		}
+	} 
+
+	if (compat.shaderLanguage == ShaderLanguage::GLSL_VULKAN && useStereo) {
+		extensions.push_back("#extension GL_EXT_multiview : enable");
```

Comment:
Won't `Preamble` write this now?  I assume we end up with two?

-[Unknown]

Reply:
Removing.